docker/erigon/config.toml

```diff
@@ -0,0 +1,16 @@
+"datadir" = "/home/erigon"
+"networkid" = "1337"
+"miner.gaslimit" = "0x1C9C380"
+"http" = true
+"http.addr" = "0.0.0.0"
```

Comment:
Can we get rid of the quotes and use sections instead?

Reply:
I tried to use sections like what we did in Geth config.toml but it didn't work, seems like erigon requires this
```
"miner.gaslimit"  = "0x1C9C380"
```
and rejects
```
[miner]
gaslimit = "0x1C9C380"
```